src/utils/docs.jl

```diff
@@ -111,9 +111,19 @@ function open_doc()
     end
 end
 
+function start_doc_preview_server(;open_browser::Bool = true, port::Int = 8000)
+  build_dir = normpath(Oscar.oscardir, "docs", "build")
+  cmd = "using Pkg; Pkg.activate(temp = true);
+         Pkg.add(\"LiveServer\"); using LiveServer;
+         LiveServer.serve(dir = \"$build_dir\", launch_browser = $open_browser, port = $port);"
```

Comment:
How about using triple quotes to avoid quoting?
```suggestion
  cmd = """
         using Pkg
         Pkg.activate(temp = true)
         Pkg.add("LiveServer")
         using LiveServer
         LiveServer.serve(dir = "$build_dir", launch_browser = $open_browser, port = $port)
         """
```

Reply:
Didn't know about this!